index.html

```diff
@@ -1,136 +1,136 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
-  <link rel="stylesheet" href="CSS/style.css" />
+  <meta charset="utf-8">
+  <title>KoalaTube EXTREME!!!</title>
+  <link rel="stylesheet" href="CSS/style.css"/>
 </head>
 <body>
 
 <div class="navbar">
   <div class="nav-container">
     <div class="nav-item-left">
-      <a href="#"><img class="logo"
+      <a href="#"><img alt="" class="logo"
           src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a8/YouTube_Rewind_Logo_%282017_to_Present%29.svg/100px-YouTube_Rewind_Logo_%282017_to_Present%29.svg.png"/>
       </a>
     </div>
     <div class="nav-item-center">
-      <a href="#">
-        <input type="text" placeholder="Search" size="80%" name="search" />
+
+        <input type="text" placeholder="Search" size="80" name="search" />
         <button type="submit" >
-          <img class="logo-s" src="https://static.thenounproject.com/png/101791-200.png"/>
+          <img alt="" class="logo-s" src="https://static.thenounproject.com/png/101791-200.png"/>
         </button>
-      </a>
+
     </div>
     <div class="nav-item-right">
       <a href="#">
-        <img class="logo" src="https://upload.wikimedia.org/wikipedia/commons/thumb/f/f2/OOjs_UI_icon_upload-ltr.svg/600px-OOjs_UI_icon_upload-ltr.svg.png"/>
+        <img alt="" class="logo" src="https://upload.wikimedia.org/wikipedia/commons/thumb/f/f2/OOjs_UI_icon_upload-ltr.svg/600px-OOjs_UI_icon_upload-ltr.svg.png"/>
       </a>
       <a href="#">
-        <img class="logo" src="https://cdn1.iconfinder.com/data/icons/business-users/512/circle-512.png"/>
+        <img alt="" class="logo" src="https://cdn1.iconfinder.com/data/icons/business-users/512/circle-512.png"/>
       </a>
-      <a href="#">
+
         <button type="button" id="sign-in">
         <span>sign in</span>
-      </button></a>
+      </button>
     </div>
   </div>
 </div>
 
 <div class="grid-container">
```

Comment:
`OPTIONAL` 

Consider using `main` instead, this shows the dominant content of the `<body>` of a document. More on `<main>` [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) :star2: